### DISCUSSION TITLE
add a toggle/config to support setting alert view mode(grouped/list/state)

### DISCUSSION THREAD

**Discussion by bossjie** [grafana/grafana#66712] [Alerting]
Default mode is grouped but they are really complex to pick up. 
we'd like to set it as list mode as default mode. 

![image](https://user-images.githubusercontent.com/8426828/232658910-a700575c-6ad2-4694-a2a1-16ae6c0bd06e.png)
![image](https://user-images.githubusercontent.com/8426828/232658982-b9465e0e-0881-4215-a971-a104f46b42ad.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82736. Feel free to continue the discussion around this there. Thank you!
